shared-module/audiomixer/Mixer.c

```diff
@@ -103,11 +103,28 @@ void audiomixer_mixer_reset_buffer(audiomixer_mixer_obj_t *self,
 
 __attribute__((always_inline))
 static inline uint32_t add16signed(uint32_t a, uint32_t b) {
+    #if (defined(__ARM_ARCH_7EM__) && (__ARM_ARCH_7EM__ == 1))  // Cortex-M4 w/FPU
```

Comment:
Let's remove the M4 comment because the M7 is the same architecture.

```suggestion
    #if (defined(__ARM_ARCH_7EM__) && (__ARM_ARCH_7EM__ == 1))
```

Reply:
Sorry, please do the rest in the file too.

Reply:
Sorry that's what I get for trying to do it late at night :(